visualime/visualize.py

```diff
@@ -360,14 +360,9 @@ def mark_boundaries(
     """
     rgb_color = _get_color(color, opacity)[:3]
 
-    # check if the shape of the mask fits the image
     if image.shape[:2] != segment_mask.shape:
         raise ValueError("The shape of the mask does not match the shape of the image.")
```

Comment:
```suggestion
        raise ValueError(f"The shape of the mask ({segment_mask.shape}) does not match "
                         f"the shape of the image ({image.shape[:2]})"
````

So etwas ist immer eine gute Unterstützung beim Debuggen :)